lib/send_text.rb

```diff
@@ -0,0 +1,25 @@
+require 'twilio-ruby'
+require 'dotenv/load'
+
+class SendText
```

Comment:
Ideally classes should be named nouns, and methods verbs. Could this be `TextSender` or `MessageHandler` or equivalent instead?